custom_components/bmw_connected_drive/sensor.py

```diff
@@ -28,10 +32,17 @@
     "remaining_range_fuel": ["mdi:map-marker-distance", LENGTH_KILOMETERS],
     "max_range_electric": ["mdi:map-marker-distance", LENGTH_KILOMETERS],
     "remaining_fuel": ["mdi:gas-station", VOLUME_LITERS],
-    "charging_time_remaining": ["mdi:update", TIME_HOURS],
-    "charging_status": ["mdi:battery-charging", None],
-    # No icon as this is dealt with directly as a special case in icon()
-    "charging_level_hv": [None, PERCENTAGE],
+    # LastTrip attributes
```

Comment:
I now realize that using `attributes` in this code is not the most logic (refering to the general code) when it has to do with HA sensors and not the attributes as used in HA.